projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php

```diff
@@ -439,6 +448,7 @@ public function render_block( $attributes ) {
 		$block_attributes   = array(
 			'headline'        => isset( $attributes['headline'] ) ? $attributes['headline'] : null,
 			'show_thumbnails' => isset( $attributes['displayThumbnails'] ) && $attributes['displayThumbnails'],
+			'show_author'     => isset( $attributes['displayAuthor'] ) ? (bool) $attributes['displayAuthor'] : true,
```

Comment:
It seems author is not shown in the current stable version, I would prefer the behaviour to be consistent with the old block - i.e. set default to `false` 😄 . Please ignore this comment if this is intended.